### ISSUE TITLE
1.4 Light client ( To be discussed )

### ISSUE THREAD

**Issue by MrishoLukamba** [OrdumLTD/Technology-Research#5]
- [ ] Work closely with Phala Team and getting feedback.
- [ ] Work with front end team on wallet infrastructure @ivo0nikolaev @martinloesethjensen 
- [ ] Issue report on how light-client improves user experience by doing user survey.

**Comment by XyloDrone** [comment]
@MrishoLukamba do you mind writing up a summary why we won't be using Phala light clients?

**Comment by MrishoLukamba** [comment]
Yeah so the main issue is phala uses light clients for their offchain workers state provability.

What we wanted to build our own light client and connect to multiple chain including phala but the issue is. 

Building a light client is one thing and connecting the light client to the live chain is another thing.

Phala must accept connecting to foreign light clients node request.

So at the moment we kinda dont need it and we need phala team and ordum team cooperation on this. And its not a vital tool
